libmscore/fraction.cpp

```diff
@@ -21,22 +21,24 @@ namespace Ms {
 //    greatest common divisor
 //---------------------------------------------------------
 
-static int gcd(int a, int b)
+static inline int gcd(int a, int b)
```

Comment:
`inline` doesn't make sense here actually. This keyword does not influcence function call inlining by a compiler, and actual inlining depends on compiler settings. A compiler should anyway be able to inline this function calls since it is defined in the same translation unit as those functions that use it (and before them).

Reply:
But would it make sense, and if so would you recommend it, for `ticks()`?

Reply:
Giving a possibility for a compiler to inline functions would certainly make sense for something used very often, but, as @shoogle already [noted](https://github.com/musescore/MuseScore/pull/4699#issuecomment-464810010), it should be done by moving its definition to a header file rather than by simply prepending `inline` keyword.